app/src/main/kotlin/com/wire/android/ui/home/appLock/forgot/ForgotLockCodeResetDeviceDialog.kt

```diff
@@ -90,23 +96,25 @@ fun ForgotLockCodeResetDeviceDialog(
             state = if (!isResetDeviceEnabled) WireButtonState.Disabled else WireButtonState.Error
         )
     ) {
-        // keyboard controller from outside the Dialog doesn't work inside its content so we have to pass the state
-        // to the dialog's content and use keyboard controller from there
-        keyboardController = LocalSoftwareKeyboardController.current
-        WirePasswordTextField(
-            state = when {
-                !isPasswordValid -> WireTextFieldState.Error(stringResource(id = R.string.remove_device_invalid_password))
-                else -> WireTextFieldState.Default
-            },
-            value = backupPassword,
-            onValueChange = {
-                backupPassword = it
-                onPasswordChanged(it)
-            },
-            autofill = false,
-            keyboardActions = KeyboardActions(onDone = { keyboardController?.hide() }),
-            modifier = Modifier.padding(bottom = dimensions().spacing16x)
-        )
+        if (!isPasswordNotRequired) {
```

Comment:
Following the comment above, we won't read this as a double negative `is not isPasswordRequired` but rather as `is not isPasswordRequired` but as following the logic just `if (isPasswordRequired) {..}` ?